mycpp/control_flow_pass.py

```diff
@@ -236,3 +351,73 @@ def visit_try_stmt(self, o: 'mypy.nodes.TryStmt') -> T:
             for t, v, handler in zip(o.types, o.vars, o.handlers):
                 with try_ctx.AddBranch(try_block.exit):
                     self.accept(handler)
+
+    def visit_del_stmt(self, o: 'mypy.nodes.DelStmt') -> T:
+        self.accept(o.expr)
+
+    # Expressions
+
+    def visit_member_expr(self, o: 'mypy.nodes.MemberExpr') -> T:
+        self.accept(o.expr)
+
+    def visit_yield_expr(self, o: 'mypy.nodes.YieldExpr') -> T:
+        self.accept(o.expr)
+
+    def visit_op_expr(self, o: 'mypy.nodes.OpExpr') -> T:
```

Comment:
I think a lot of these methods can be left out, since the default in

~/wedge/oils-for-unix.org/pkg/mypy/0.780/mypy/traverser.py

does the same thing

I guess that is sorta nice about the visitor compared to switch

Although I think the visitor is worse in other cases

Reply:
Oh, cool. That's convenient. I'll remove the ones that don't have any extra logic.